# Microbial_genomic
Awesome tools for ðŸ¦ 

# Bioinformatics Tools Repository

This repository contains a curated list of bioinformatics tools commonly used in microbial genomic analyses and other scientific research. Each tool listed below includes its version number and a brief description.

## Available Tools

### [Miniconda] (https://github.com/kaust-rccl/ibex-miniconda-install.git) 
**Description**: A free minimal installer for conda, a package management and environment management system.

### BLAST
**Description**: Basic Local Alignment Search Tool for comparing nucleotide and protein sequences.


### Unicycler
**Version**: 0.5.0  
**Description**: Hybrid assembler for bacterial genomes, combining Illumina and long reads for improved assembly.

### QUAST
**Version**: 5.2.0  
**Description**: Quality Assessment Tool for genome assemblies, evaluating quality with various metrics.

### Prokka
**Version**: 1.14.6  
**Description**: Rapid annotation of prokaryotic genomes to produce GFF3, GBK, and FNA files.

### Bakta
**Version**: 1.9.3  
**Description**: Rapid and standardized annotation tool for microbial genomes.

### Snippy
**Version**: 4.4.3  
**Description**: Rapid haploid variant calling and core genome alignment.

### MetaPhlAn
**Version**: 4.0.6  
**Description**: Metagenomic Phylogenetic Analysis, a tool for profiling microbial communities.

### [FSM-lite](https://github.com/nvalimak/fsm-lite.git)
**Version**: 1.0  
**Description**: Frequency-based string mining tool for efficient and accurate DNA sequence analysis.

### Scoary
**Version**: 1.6.16  
**Description**: Tool for the discovery of statistically significant bacterial gene traits.

### Gubbins
**Version**: 3.3.2  
**Description**: Detects recombination events in bacteria genomes.

### SNP-sites
**Version**: 2.5.1  
**Description**: Rapid extraction of SNP sites from multi-FASTA alignments.

### MLST
**Version**: 2.23.0  
**Description**: Multi-Locus Sequence Typing for bacterial identification and characterization.

### SRST2
**Version**: 0.2.0  
**Description**: Sequence-based typing of genes, alleles, and multi-locus sequence types.

### AMRFinderPlus
**Version**: 3.11  
**Description**: Tool for finding antimicrobial resistance genes in bacterial sequences.

### BEAST
**Versions**: 2.7.5, 1.10.4  
**Description**: Bayesian Evolutionary Analysis by Sampling Trees, a software package for evolutionary analysis.

### SeqTK
**Version**: 1.4  
**Description**: A toolkit for processing sequences in the FASTA or FASTQ format.

### Panaroo
**Version**: 1.3.4  
**Description**: A robust, scalable tool for pan-genome analysis of closely related prokaryotes.

### Guppy
**Version**: 6.1.7  
**Description**: Basecalling software for Oxford Nanopore sequencing data.

### Staphopia-sccmec
**Version**: v1.0.0  
**Description**: A pipeline for identifying the staphylococcal cassette chromosome mec (SCCmec) elements.

### spa_typing
**Version**: 0.1.0  
**Description**: Software for determining the spa type of Staphylococcus aureus strains from sequence data.

### AgrVATE
**Version**: v1.0.2  
**Description**: A tool for typing the accessory gene regulator (agr) system in Staphylococcus aureus.

### Iris
**Version**: 0.9.7  
**Description**: Visual exploration platform for genomic datasets.

### Bactopia
**Version**: v3.0.1  
**Description**: A flexible pipeline for complete analysis of bacterial genomes.

### FastTree
**Version**: 2.1.11  
**Description**: Approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences.


